content/en/docs/concepts/security/index.md

```diff
@@ -630,333 +631,277 @@ runtime. When a request comes to the proxy, the authorization engine evaluates
-binding custom resources and the policies set within completely.
-
-Assuming you have a MongoDB service on port 27017, the following example configures a service role and
-a service role binding to only allow the `bookinfo-ratings-v2` in the Istio mesh to access the
-MongoDB service.
+To allow only **authenticated** users, set `principal` to `"*"` instead, for example:
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRole
-metadata:
-  name: mongodb-viewer
-  namespace: default
-spec:
-  rules:
-  - services: ["mongodb.default.svc.cluster.local"]
-    constraints:
-    - key: "destination.port"
-      values: ["27017"]
----
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRoleBinding
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: bind-mongodb-viewer
-  namespace: default
+ name: httpbin
+ namespace: foo
 spec:
-  subjects:
-  - user: "cluster.local/ns/default/sa/bookinfo-ratings-v2"
-  roleRef:
-    kind: ServiceRole
-    name: "mongodb-viewer"
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["*"]
+   to:
+   - operation:
+       methods: ["GET", "POST"]
 {{< /text >}}
 
-### Authorization permissive mode
-
-The authorization permissive mode is an experimental feature in Istio's 1.1 release. Its interface can change in future releases.
-
-The authorization permissive mode allows you to verify authorization policies
-before applying them in a production environment.
+### Using Istio authorization on plain TCP protocols
 
-You can enable the authorization permissive mode on a global authorization
-configuration and on individual policies. If you set the permissive mode on a global
-authorization configuration, all policies switch to the permissive mode regardless
-of their own set mode. If you set the global authorization mode to
-`ENFORCED`, the enforcement mode set by the individual policies takes effect.
-If you do not set a mode, both the global authorization configuration and the individual
-policies are set to the `ENFORCED` mode by default.
+Istio authorization supports workloads using any plain TCP protocols, such as MongoDB. In this case,
+you configure the authorization policy in the same way you did for the HTTP workloads.
+The difference is that certain fields and conditions are only applicable to HTTP workloads.
+These fields include:
 
-To enable the permissive mode globally, set the value of the `enforcement_mode:` key in the global Istio RBAC authorization configuration to `PERMISSIVE` as shown in the following example.
+- The `request_principals` field in the source section of the authorization policy object
+- The `hosts`, `methods` and `paths` fields in the operation section of the authorization policy object
+- The supported conditions are listed in the [conditions page](/docs/reference/config/authorization/conditions/).
```

Comment:
- The supported conditions ~are~ listed ~i~on the [conditions page](/docs/reference/config/authorization/conditions/).